ethereum/env/.env.seievm

```diff
@@ -0,0 +1,13 @@
+# Sei EVM mainnet read only env. Use to deploy the core contract with forge.
+
+# ethereum$ ln -s env/.env.seievm .env
+
+RPC_URL=""
+FORGE_ARGS="--legacy --with-gas-price 3000000000" # 3 gwei
+
+# Wormhole Core
+INIT_SIGNERS=["0x58CC3AE5C097b213cE3c81979e1B9f9570746AA5"]
+INIT_CHAIN_ID=40
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=713715
```

Comment:
this is the chain ID for Sei Devnet - https://chainlist.org/chain/713715
should this file be `.env.seievm.testnet`?